### DISCUSSION TITLE
Correct settings to use the Gemini 2.3 validations

### DISCUSSION THREAD

**Discussion by howaskew** [AstunTechnology/os-custom-geonetwork#4] [Q&A]
In Admin Console / Metadata and Templates / Validation
When I click on one of the iso19139.gemini23 validations, they are all shown as 'This criteria always passes'

![image](https://user-images.githubusercontent.com/19271569/105364379-02fa3800-5bf5-11eb-90fb-4d3b5336f6bd.png)

Is this correct? 

I'm not sure how the schematrons are being applied but Edd noticed that 2.2 valid records were being imported even when he requested 2.3 validations - could that be to do with this setting?

**Comment by archaeogeek** [comment]
Hi Howard,

This is a two-parter...

1) The schematrons that a record gets validated against are determined by the schema assigned to the record when it's imported. See https://github.com/AstunTechnology/iso19139.gemini23/wiki/Ensuring-your-metadata-records-are-recognised-as-Gemini-2.3 for the values a record needs to have to be automatically recognised as Gemini 2.3. If you apply the Gemini 2.3 transformation when importing, then Geonetwork will upgrade the record to include those values and ensure that it gets assigned the correct schema. If Edd didn't explicitly choose the transformation, or he didn't include the necessary information then it probably wouldn't work.  If he thinks there's a problem with his setup then he should probably submit an issue :-)

2) The above settings in the image are to do with how it reports validation. If set as above, it means that the record must pass all validation rules in that schematron to be marked as valid in the catalog. You can set "only report errors" rather than "required to be valid" if you want errors to be reported as warnings instead. I leave that to the client to decide which approach they want, but my preference is to choose "required to be valid" for the main Gemini 2.3 schematron and "only report errors" for the Gemini 2.3 supplemental schematron.

Hope that makes sense?

Jo

**Reply by howaskew** [reply]
Thanks Jo - and on number 2 what's your preference for the final schematron:  schematron-rules-ISOTS19139A1Const?

**Reply by archaeogeek** [reply]
It depends- personally I'd set it as "required to be valid" as it is used on data.gov.uk

**Reply by PeterParslow** [reply]
I agree with Jo. Actually, I think the GEMINI schematron(s) assume that the file is valid to that one, as it is part of confirming that it is valid ISO 19139 testing the parts that XSD can't reach.
